src/main/kotlin/no/nav/familie/ef/iverksett/brev/DistribuerVedtaksbrevTask.kt

```diff
@@ -17,17 +18,69 @@ import java.util.UUID
                      triggerTidVedFeilISekunder = 15 * 60L,
                      beskrivelse = "Distribuerer vedtaksbrev.")
 class DistribuerVedtaksbrevTask(private val journalpostClient: JournalpostClient,
-                                private val tilstandRepository: TilstandRepository) : AsyncTaskStep {
+                                private val tilstandRepository: TilstandRepository,
+                                private val featureToggleService: FeatureToggleService) : AsyncTaskStep {
 
     private val logger: Logger = LoggerFactory.getLogger(this::class.java)
 
     override fun doTask(task: Task) {
         val behandlingId = UUID.fromString(task.payload)
+
+        validerJournalpostresultat(behandlingId)
+
+        val vedtaksbrevHarEnkelMottaker = tilstandRepository.hentJournalpostResultat(behandlingId) !== null
```

Comment:
Trenger vi egentlige 2 kolonner for om det er til en eller flere personer? 
Hadde vi kunnet oppdatert kolonnen med kun 1 verdi til en map i stedet? 
Tenker koden blir generellt litt enklere då

Gjelder både journalPostResultat og DistribuerVedtaksbrevResultat

Reply:
Godt innspill, prøver på det :)

Reply:
Synes denne ser veldig lovende ut - hvis vi får "schmakket" i hop dette så vil det jo være supersmooth!